### DISCUSSION TITLE
What is stored outside of the default storage location?

### DISCUSSION THREAD

**Discussion by anaxonda** [osmandapp/OsmAnd#18595] [Q&A]
I'm hoping for some clarification about what is stored outside the default storage location. 
If I chose my external sd card as the osmand storage location, and try to move the sd card to a different phone, some things are not preserved: map markers and app settings (and maybe other stuff?).

If I change my storage location from the custom location on the sd card to one of the default locations, my favorites are still preserved.

Can anyone explain how this works, and how can I move settings to a new device (without using cloud backup)?

Thanks

**Comment by DmitryAlexei** [comment]
You can back up all your data using Export feature (see https://osmand.net/docs/user/personal/import-export#export)
Go to menu > settings > and choose *Export to file* option

**Comment by anaxonda** [comment]
Do you know where these data are stored? In the private app storage on the (internal) sd card?

Why are they stored separately from everything else (maps, favorites, etc)?

**Comment by anaxonda** [comment]
Thinking about this again...

Obviously users can use the export function, or use Osmand Cloud to sync data. It would be really nice however just to use syncthing to sync directories between devices, or be able to move external sd card and be good to go. Is it worth filing an issue for this idea?

**Reply by anaxonda** [reply]
Just saw discussion #12194, similar question.
